go_utils/photo_download.py

```diff
@@ -380,27 +397,13 @@ def get_lc_download_targets(  # noqa: C901
     """
     arguments = locals()
     targets = set()
-    num_invalid_photos = 0
+    num_invalid_photos = {"count": 0}
```

Comment:
What's the advantage of keeping it in a dict?

Reply:
Python doesn't allow modifications to an outer variable from within a nested function, so I found a workaround by creating a 1-element dict of the num_invalid_photos -- Here's a [source](https://stackoverflow.com/questions/11987358/why-nested-functions-can-access-variables-from-outer-functions-but-are-not-allo).

Reply:
Would it be possible to return two things (targets, num invalid) from that function and increment the variable that way?

Reply:
Sure -- we could return `num_invalid_photos` from the nested function and then sum the vectorized output from `get_mhm/lc_photo_args`.

Alternatively, with the dictionary, we can provide the User with more details on the types of photos skipped (see new commit) with messages like `Skipped 5 invalid photos: {'invalid_URL': 3, 'rejected': 1, 'pending': 1, 'bad_photo_id': 0}`. This could help the user better understand their dataset.

Reply:
I think the latter would be really nice. So we can keep the dictionary as long as we do it like that

Reply:
Cool beans! -- It's implemented in the most recent commit :)